src/HasEnums.php

```diff
@@ -28,6 +30,86 @@ public function getAttribute($key)
             : $value;
     }
 
+    /**
+     * @param Builder $builder
```

Comment:
We usually use fqcn in docblocks

Reply:
https://github.com/spatie/laravel-enum/pull/4/commits/4aa4a0b6e341290b295db81dbe47807b8b346465